nixos/modules/config/networking.nix

```diff
@@ -187,11 +199,11 @@ in
         "rpc".source = pkgs.glibc.out + "/etc/rpc";
 
         # /etc/hosts: Hostname-to-IP mappings.
-        "hosts".text =
+        "hosts".text = let foo = concatStringsSep " " cfg.extraLocalHosts; in
```

Comment:
What my predecessor meant to say, is that `foo` is in general not a good name for a variable as it is too generic.